Server/src/loaders/loadStartupProcess.ts

```diff
@@ -49,21 +49,41 @@ export class loadStartupProcess {
     app.use('/', fileUploadRouter)
 
 
+
+    const config: any = {
+
+      "type": process.env.DB_TYPE,
+      "host": process.env.HOST,
+      "port": process.env.DB_PORT,
+      "username": process.env.USERNAME,
```

Comment:
I like that we store this in our local env files, how ever add **/env to gitignore so that noone commits it by accident

Reply:
now that the gitignore was commited this is resolved

Reply:
ok now that the env is in .env also add .env to .gitignore